idb/src/Splash.js

```diff
@@ -1,19 +1,56 @@
 import React, { Component } from 'react';
+import Slider from 'react-slick';
 import logo from './logo.svg';
+import pic1 from './assets/pic1.jpg'
+import pic2 from './assets/pic2.jpg'
+import pic3 from './assets/pic3.jpg'
+import header from './assets/header.png'
 import './App.css';
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
 
 
 export default class Splash extends Component {
+  constructor(props) {
+    super(props);
+
+    this.settings = {
+      dots: true,
+      infinite: true,
+      speed: 300,
+      arrows: true,
+      slidesToShow: 1,
+      slidesToScroll: 1,
+      autoplay: true,
+      centerMode: true,
+      autoplaySpeed: 4000,
+      pauseOnHover: true
+    };
+  }
   render() {
+
     return (
       <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
+      <header className="App-header">
+          <img src={header} className="App-logo" alt="logo" />
         </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
+      <div className="carousel-parent">
+          <Slider {...this.settings} className="carousel">
```

Comment:
Consider an approach where you render the elements before the return function for performance improvements

Reply:
Fixed by https://github.com/WeTheSWEople/SWEThePeople/pull/26/commits/f31cee3e26a13015028a7269a146e3751633a12c